src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const employees = document.querySelectorAll('li[data-position]');
+const ul = document.querySelector('ul');
+
+function sortList(list) {
+  return Array.from(list).sort(
+    (employee1, employee2) =>
+      parseFloat(employee2.dataset.salary.replace(/[^0-9.-]+/g, '')) -
+      parseFloat(employee1.dataset.salary.replace(/[^0-9.-]+/g, '')),
+  );
+}
+
+function getEmployees(list) {
+  return list
```

Comment:
The `list` parameter in `getEmployees` is expected to be an array to use the `map` method. However, `employees` is a NodeList, which does not have a `map` method. Consider converting the NodeList to an array using `Array.from(list)` before calling `map`.